### PR TITLE
Code refactoring - Removed unnecessary usings, added regions and comments

### PR DESCRIPTION
**Pull request by gauravgupta98**

### CODE REVIEW

File: lib/PuppeteerSharp/ContextPayload.cs [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
+        /// <summary>
+        /// Gets or sets the context payload aux data.
+        /// </summary>
         public ContextPayloadAuxData AuxData { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name.
+        /// </summary>
         public string Name { get; set; }
```
Comment (kblok) [inline lib/PuppeteerSharp/ContextPayload.cs:18]: We are not adding XML Docs for internal classes.

File: lib/PuppeteerSharp/EmulationManager.cs [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
 
 namespace PuppeteerSharp
 {
     internal class EmulationManager
     {
+        #region Members
         private readonly CDPSession _client;
         private bool _hasTouch;
         private bool _emulatingMobile;
+        #endregion Members
```
Comment (kblok) [inline lib/PuppeteerSharp/EmulationManager.cs:13]: I'm trying to get out of the `#region` blocks. The community is not using them anymore.

**Review by kblok — CHANGES_REQUESTED** [review]
Thanks for taking the time to improve the code.
I like removing using, but I don't think we need XML docs for the internals classes. And it seems that the community stop using regions so I might end up removing them.

File: lib/PuppeteerSharp/ContextPayload.cs (hunk 1 above)
Comment (gauravgupta98) [reply lib/PuppeteerSharp/ContextPayload.cs:18]: Thanks for the review @kblok.

File: lib/PuppeteerSharp/EmulationManager.cs (hunk 2 above)
Comment (gauravgupta98) [reply lib/PuppeteerSharp/EmulationManager.cs:13]: Wasn't aware of this. Thanks for letting know :)

**Comment by gauravgupta98** [comment]
Closing the PR as we don't need the suggested changes

**Pull request closed without merging**